### ISSUE TITLE
Generated file has errors when there are two classes with same name in different packages

### ISSUE THREAD

**Issue by EnilPajic** [vert-x3/vertx-codegen#194]
Hi!

My EB service looks like this:
```java
@Fluent
MyService getTickets1(JsonObject whatever, Handler<AsyncResult<List<Ticket>>> e);

@Fluent
MyService getTickets2(JsonObject whatever, Handler<AsyncResult<List<package2.Ticket>>> e);
```
notice that handlers accept lists of `Ticket`s from two different packages (`package1` which is imported (`import package1.Ticket;`) and `package2`).

When I try to build my project, generated file (`MyServiceVertxEBProxy`) have two ambiguous imports:
```java
import package1.Ticket;
import package2.Ticket;
```

And of course Java cannot distinguish which `Ticket` to use and throws compile time error on the second import "_a type with the same simple name is already defined by the single-type-import of package1.Ticket_"

Of course, this is simplified example. I'm using `vertx-codegen` version `3.5.0`.

**Comment by EnilPajic** [comment]
Hello, anything on this? (Ping comment)

**Comment by vietj** [comment]
nobody has been looking at this issue, actually nobody has examined yet how this should be handled best

**Comment by EnilPajic** [comment]
Aha, thank you. I'm sorry to hear that, this is a such useful library. 
I assume the simplest solution would be to use full qualified names (e.g. with package) on every place, but I'm not fully competent.

Feel free to close the issue then.

**Comment by vietj** [comment]
yes that's a solution, but it means that the generators shall be rewritten and it's not the codegen lib that is faulty.

**Comment by vietj** [comment]
in your case the service-proxy generator.
